test/unit/frontend.d

```diff
@@ -71,3 +71,48 @@ unittest
 
     assert(visitor.created);
 }
+
+@("initDMD - contract checking")
+unittest
+{
+    import std.algorithm : each;
```

Comment:
considering the following comment, use foreach() to prevent pulling phobos in this unittest

Reply:
Phobos is already used: https://github.com/dlang/dmd/blob/b0007c1193dfb72bcfa8cef3f420fee68564ce49/src/dmd/frontend.d#L21-L23

Reply:
Yes okay but I had the feeling that it was avoided, following a message on the NG.
Anyway, I'm not blocking this PR. Simple suggestion.

Reply:
> Yes okay but I had the feeling that it was avoided

Yes, in general. But the tests in `test/unit` is intended to use DMD as a library and is a bit different than the rest of the tests.

Reply:
DMD as a library can and should use Phobos. No point in reinventing the wheel here.